src/generator/AutoRest.Ruby.Azure/CodeGeneratorRba.cs

```diff
@@ -73,8 +72,11 @@ public CodeGeneratorRba()
                     continue;
                 }
 
-                var modelTemplate = new ModelTemplate { Model = model };
-                await Write(modelTemplate, Path.Combine(GeneratorSettingsRb.Instance.modelsPath, CodeNamer.UnderscoreCase(model.Name) + ImplementationFileExtension));
+                var modelTemplate = new AzureModelTemplate { Model = model };
```

Comment:
I liked the idea that @veronicagg has but looks like you guys already discussed and reached to mutual conclusion